.travis.yml

```diff
@@ -22,10 +27,33 @@ script:
 notifications:
   email: false
 after_success:
-- |
-  if [[ $TRAVIS_BRANCH == 'master' && $TRAVIS_PULL_REQUEST == 'false' ]]; then
-    curl "$JENKINS_DEPLOY_URL"
-  fi
-  if [[ $TRAVIS_TAG != "" ]]; then
-    curl "$JENKINS_TAG_DEPLOY_URL"
-  fi
+  - |
+    curl https://api.ipify.org; echo
```

Comment:
Looks like this line is for discovering the Travis system's IP address? If this is needed, any chance you can provide an in-line comment that describes how it's being used?

Reply:
This is canary to verify that our firewall matches what Travis is using. We had an incident in the past where our firewall was out of date. This way we can verify that our firewall is, or is not, up to date.